### PR TITLE
Assistant update

### PR DESCRIPTION
**Pull request by kevinkowa**
Adding new function to allow null values on properties https://github.ibm.com/Watson/developer-experience/issues/9310

### CODE REVIEW

File: assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNode.java [hunk 1]
```diff
@@ -0,0 +1,724 @@
+/*
```
Comment (apaparazzi0329) [inline assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNode.java:1]: Why was this entire file created? Did it not exist before

File: assistant/src/test/java/com/ibm/watson/assistant/v1/model/UpdateDialogNodeTest.java [hunk 2]
```diff
@@ -0,0 +1,366 @@
+/*
```
Comment (apaparazzi0329) [inline assistant/src/test/java/com/ibm/watson/assistant/v1/model/UpdateDialogNodeTest.java:1]: Same question with this file? Did it not exist before

File: assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNode.java (hunk 1 above)
Comment (kevinkowa) [reply assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNode.java:1]: it's generated with the merge-patch

File: assistant/src/test/java/com/ibm/watson/assistant/v1/model/UpdateDialogNodeTest.java (hunk 2 above)
Comment (kevinkowa) [reply assistant/src/test/java/com/ibm/watson/assistant/v1/model/UpdateDialogNodeTest.java:1]: it's a unit test generated by the merge-patch

**Review by mediumTaj — APPROVED** [review]
👍  looks good! Be sure to document this as handwritten change

File: assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNode.java (hunk 1 above)
Comment (kevinkowa) [reply assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNode.java:1]:
```java
UpdateDialogNode updateDialogNode = new UpdateDialogNode.Builder()
              .description(dialogNodeDescription2)
              .nextStep(null)
              .dialogNode(dialogNodeName2)
              .build();
      Map<String, Object> body = updateDialogNode.asPatch();

      UpdateDialogNodeNullableOptions updateDialogNodeNullableOptions =
              new UpdateDialogNodeNullableOptions.Builder()
                      .workspaceId(workspaceId)
                      .dialogNode(dialogNodeName)
                      .body(body)
                      .build();
```
UpdateDialogNode contains the parameters that you can change/update
UpdateDialogNodeNullableOptions contains the parameters needed for the service call & the variables that were set in the UpdateDialogNode Model as a map

Comment (apaparazzi0329) [reply assistant/src/main/java/com/ibm/watson/assistant/v1/model/UpdateDialogNode.java:1]: Ok this makes sense. We will have to be clear in the documentation on how to use this new method

**Review by apaparazzi0329 — APPROVED** [review]
LGTM

**Review by repjarms — APPROVED** [review]

**Pull request merged**